app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -445,6 +499,57 @@ fun AddItemDialog(
     }
 }
 
+@Composable
+fun DeleteItemDialog(
```

Comment:
![image](https://user-images.githubusercontent.com/69108816/207669535-5ceb4ab0-b6b6-43ce-a85f-8c19ef173699.png)

Nie wydaje ci się ze te przyciski są trochę za duże w stosunku do tekstu? Więc myślę ze albo można zwiększyć tekst albo zmniejszyć przyciski. Przy czym to drugie to będzie możliwe dopiero jak zmergujemy mój refaktor buttona

Reply:
tak, nieco słabo to wygląda, chciałem jakąś inną czcionkę tu ustawić, więc jeśli się uda to zmienię czcionkę 
(i powiększę napis), a jeśli nie to po prostu zmniejszę przyciski